image-make

```diff
@@ -131,18 +131,18 @@ for tag in "${TagArray[@]}";do
     Image=$RepoName/$DockerName:$tag
     echo "Image $Image"
     if [ -z "$BuiltImage" ];then
-	set -x
+        set -x
         docker build -t $Image $DockerfileDir
-	set +x
-	BuiltImage=$Image
+        set +x
+        BuiltImage=$Image
     fi
     set -x
     docker tag $BuiltImage $ZANATA_DOCKER_PUSH_REGISTRY/$Image
     set +x
     if [ $IsPush -eq 1 ];then
-	set -x
-	docker push $ZANATA_DOCKER_PUSH_REGISTRY/$Image
-	set +x
+        set -x
+        docker push $ZANATA_DOCKER_PUSH_REGISTRY/$Image
```

Comment:
what about authentication against the Docker registry? We might need to either mention that needs to be done beforehand, or include it somehow in the script